### PR TITLE
Added ability to enable/disable individual resources

### PR DESCRIPTION
**Pull request by thyseus**
Added ability to enable/disable individual resources

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to `sky` will be documented in this file
 
+## 2.1.4 - to be releasd
```
Comment (atmonshi) [inline CHANGELOG.md:5]: you can revert these changes in this file since it is automatically generated when making a new release.
and add this to the description of the PR

File: config/zeus-sky.php [hunk 2]
```diff
@@ -23,9 +23,14 @@
...
 
     /**
-     * enable or disable FAQ addons.
+     * enable or disable individual Resources.
      */
-    'enableFaq' => true,
+    'enabled_resources' => [
+        LaraZeus\Sky\Filament\Resources\PostResource::class,
+        LaraZeus\Sky\Filament\Resources\PageResource::class,
+        LaraZeus\Sky\Filament\Resources\TagResource::class,
```
Comment (atmonshi) [inline config/zeus-sky.php:31]: we can't disable tags since it will break other resources

File: CHANGELOG.md (hunk 1 above)
Comment (thyseus) [reply CHANGELOG.md:5]: Ok, will do.

File: config/zeus-sky.php (hunk 2 above)
Comment (thyseus) [reply config/zeus-sky.php:31]: Yes, i see. I will make 'tags' being always active and it is just being hidden in the Navigation Bar then, give me a second...

**Comment by atmonshi** [comment]
thanks, @thyseus for the support, I added some comments to review

File: src/SkyServiceProvider.php [hunk 3]
```diff
@@ -49,11 +45,6 @@ public function configurePackage(Package $package): void
 
     protected function getResources(): array
     {
-        return [
-            PostResource::class,
-            PageResource::class,
-            TagResource::class,
-            FaqResource::class,
-        ];
+        return config('zeus-sky.enabled_resources');
```
[the remark this answers is not in the document]
Comment (thyseus) [reply src/SkyServiceProvider.php]: @atmonshi thanks a lot for reviewing, i adjusted the code so that TagResource is not disableable.
Whats your opinion ?

**Comment by atmonshi** [comment]
one last thing, would you update the configuration page in the docs, the new config option

https://github.com/lara-zeus/sky/blob/main/docs/configuration.md

thank you.

**Review by atmonshi — APPROVED** [review]

**Comment by atmonshi** [comment]
Thanks, @thyseus , I'll realesse this soon.

TBH I only add featuers as needed, so there are no plans for full text search, go ahead and thank you on advance :)
